.github/workflows/deploy-book-ghpages.yml

```diff
@@ -100,10 +100,14 @@ jobs:
         cat book/_toc.yml
         jupyter-book build book/
 
+    - name: Clean branch name of disallowed characters
```

Comment:
@severinbratus , could this clean branch name be stored or returned somewhere?  Maybe even an 'overview' page which includes all of this: https://github.com/TeachBooks/Sandbox/branches? On https://teachbooks.github.io/overview?

Reply:
@severinbratus , or a better solution! List all the published branches with their links on a 404 page which you update after the build!

Reply:
You might use / improve the general 404 page: https://teachbooks.github.io/404

Reply:
Okay @rlanzafame  and I discussed this some more. Maybe a log in the actions itself with the final urls is the best :)